docs/cli/quick-guide.md

```
@@ -0,0 +1,151 @@
+
+### 2.1. Archive the data
+
+Archive the data:
+
+```shell
+tar -czvf <DATA_NAME>.tar.gz -C <DATA_PATH> .
+```
+
+For example:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)